.github/workflows/TestLatest.yml

```diff
@@ -16,11 +16,11 @@ jobs:
   test:
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: true
+      fail-fast: false # keep other os running if one fails
       matrix:
         julia-version: ['1']
         julia-arch: [x64]
-        os: [windows-latest] # ubuntu-latest, 
+        os: [windows-latest, ubuntu-latest] # ubuntu-latest, 
```

Comment:
does this work? FMIExport only runs on windows for now

Reply:
> does this work? FMIExport only runs on windows for now

yes it does. Thats the test-action beeing ubuntu-ready already, although FMIExport ist not ready yet.